lib/phoenix_live_reload/channel.ex

```diff
@@ -16,7 +16,7 @@ defmodule Phoenix.LiveReload.Channel do
 
   def handle_info({_pid, {:fs, :file_event}, {path, _event}}, socket) do
     if matches_any_pattern?(path, socket.assigns[:patterns]) do
-      asset_type = Path.extname(path) |> String.lstrip(?.)
+      asset_type = String.replace(Path.extname(path), ~r/^\.+/,  "")
```

Comment:
We can move it to a separate function:

```elixir
def remove_leading_dot("." <> rest), do: rest
def remove_leading_dot(rest), do: rest
```

I would prefer that to using regexes. Can you please change it accordingly?

Thank you!